mono/metadata/sre-encode.c

```diff
@@ -476,7 +476,7 @@ mono_dynimage_encode_locals (MonoDynamicImage *assembly, MonoReflectionILGen *il
  * Copy len * nelem bytes from val to dest, swapping bytes to LE if necessary.
  * dest may be misaligned.
  */
-static void
+static G_GNUC_UNUSED void
```

Comment:
Can you just put the whole function between `#if G_BYTE_ORDER != G_LITTLE_ENDIAN` ?